dev-resources/puppetlabs/services/certificate_authority/certificate_authority_int_test/ca_true_test/master/conf/ssl/ca/requests/test_cert_ca_true.pem

```diff
@@ -0,0 +1 @@
+
```

Comment:
Also if this doesn't exist anymore and we just create it from inside the test, I don't care so much about the directory name - I just wanted to follow the convention for creating a directory in `dev-resources`.